app/assets/stylesheets/home.scss

```diff
@@ -75,7 +75,7 @@
         margin-top: 10px;
         font-weight: 700;
       }
-      @include btn-ror(transparent, $yellow, white);
+      @include btn-ror(white, transparent, white);
```

Comment:
The button looks a bit strange now :thinking:. Maybe using a green button, like in other parts of the web? :thinking: